fast/stages/1-resman/billing.tf

```diff
@@ -17,20 +17,38 @@
 # tfdoc:file:description Billing resources for external billing use cases.
 
 locals {
-  # used here for convenience, in organization.tf members are explicit
-  billing_ext_users = compact([
-    try(module.branch-network-sa.iam_email, null),
-    try(module.branch-pf-dev-sa.iam_email, null),
-    try(module.branch-pf-prod-sa.iam_email, null),
-    try(module.branch-pf-sa.iam_email, null),
-    try(module.branch-security-sa.iam_email, null),
-    try(module.branch-dp-dev-sa[0].iam_email, null),
-    try(module.branch-dp-prod-sa[0].iam_email, null),
-    try(module.branch-gcve-dev-sa[0].iam_email, null),
-    try(module.branch-gcve-prod-sa[0].iam_email, null),
-    try(module.branch-gke-dev-sa[0].iam_email, null),
-    try(module.branch-gke-prod-sa[0].iam_email, null)
-  ])
+  billing_iam = merge(
+    # stage 2
+    var.fast_stage_2.networking.enabled != true ? {} : {
```

Comment:
nit: I'm sure there's a reason but this condition is different to the others (`x != true` vs `!x`)